benchmarks/experiment_runner.py

```diff
@@ -414,9 +414,14 @@ def _prepare_inputs(self, example_inputs, should_randomize_input):
       inputs_list.append(inputs)
     return inputs_list
 
-  def _mark_step(self, benchmark_experiment):
+  def _mark_step(self, benchmark_experiment, tensor_to_check=None):
```

Comment:
Wouldn't it be more convenient to accept a list of tensors?

Reply:
`tensor_to_check` is already a list of tensors. Update the name to `tensors_to_check`.